meta-ostro-xt/recipes-opencl/ocl-icd/ocl-icd/configurable-doc-generation.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/configure.ac b/configure.ac
```

Comment:
Almost all git actions(atleast in this context am,apply) look for committer details before applying the patch.